contents/docs/product-analytics/identify.mdx

```diff
@@ -39,13 +38,13 @@ export const splitUsersDark = "https://res.cloudinary.com/dmukukwp6/video/upload
 
 <JSIdentifyUseUniqueIds/>
 
-### 3. Reset after logout
+### 3. Set up person profiles and properties
 
-<JSIdentifyReset/>
+<JSIdentifySetUserProperties/>
 
-### 4. Person profiles and properties
+## Reset after logout
 
-<JSIdentifySetUserProperties/>
+When a user logs out, you should [call reset](/docs/product-analytics/identify/reset) to prevent future events on the device from getting accidentally grouped with that user.
```

Comment:
```suggestion
When a user logs out, you should [call `reset()`](/docs/product-analytics/identify/reset) to prevent future events on the device from getting accidentally grouped with that user.
```